MODULE.bazel

```diff
@@ -6,14 +6,14 @@ module(
     version = "0.0.0",
 )
 
-bazel_dep(name = "aspect_bazel_lib", version = "1.29.2")
+bazel_dep(name = "aspect_bazel_lib", version = "1.30.2")
 # Note: only used for provider symbols, we don't spawn nodejs actions
 bazel_dep(name = "aspect_rules_js", version = "1.23.1")
 bazel_dep(name = "bazel_skylib", version = "1.4.1")
 bazel_dep(name = "platforms", version = "0.0.4")
 
 swc = use_extension("@aspect_rules_swc//swc:extensions.bzl", "swc")
-swc.toolchain(name = "swc", swc_version = "v1.3.41")
+swc.toolchain(name = "swc", swc_version = "v1.3.56")
```

Comment:
this affects users as well, right? I wonder if we should be registering our toolchain as dev_dependency

Reply:
https://github.com/aspect-build/rules_swc/commit/23fc373a40be33beb3670ee3f63902b20188ef53#diff-22a5b39d0ea9876499ff002558035fe01c08d13142f2120facd5c2393493d456 upgraded the default version so I thought this should align with it...?